op-e2e/blobs/blob_test.go

```diff
@@ -0,0 +1,178 @@
+package blobs
+
+import (
+	"bytes"
+	"context"
+	"crypto/ecdsa"
+	"errors"
+	"fmt"
+	"math/big"
+	mrand "math/rand"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-e2e"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/wait"
+	"github.com/ethereum-optimism/optimism/op-e2e/system/e2esys"
+	"github.com/ethereum-optimism/optimism/op-node/node"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/testutils"
+	"github.com/ethereum-optimism/optimism/op-service/txmgr"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/consensus/misc/eip4844"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethstorage/da-server/pkg/da"
+	"github.com/ethstorage/da-server/pkg/da/client"
+	"github.com/holiman/uint256"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	dacPort = 37777
+)
+
+var (
+	ctx       = context.Background()
+	dacUrl    = fmt.Sprintf("http://127.0.0.1:%d", dacPort)
+	toAddress = testutils.RandomAddress(mrand.New(mrand.NewSource(dacPort)))
+	blobs     = make([]*eth.Blob, 3)
```

Comment:
Why not just define it inside `TestFunctionSuccess` since it's only referenced in that function?